### PR TITLE
Feature/offchain events

### PR DESCRIPTION
**Pull request by OR13**
Implement string events in prep for IPFS Events

### CODE REVIEW

**Comment by OR13** [comment]
Need to add more tests for string values

**Comment by OR13** [comment]
rather than comment out tests to fix this, I will choose to figure out how to get these tests to pass on travis

**Comment by OR13** [comment]
Once i finish https://github.com/transmute-industries/transmute-framework/issues/52 this PR should be good to go

File: README.md [hunk 1]
```diff
@@ -58,4 +58,5 @@ If these fail, expect your PR to be rejected ; )
 
 - https://github.com/Upchain/truffle-template
 - https://github.com/transmute-industries/eth-faucet
-
+- https://github.com/AugurProject/augur/blob/master/src/modules/auth/actions/register.js
+- https://airbitz.co/developer-api-library/
```
Comment (eolszewski) [inline README.md]: Not really needed

File: src/CLI/EventStore/index.ts [hunk 2]
```diff
@@ -0,0 +1,124 @@
+// var path = require('path')
```
Comment (eolszewski) [inline src/CLI/EventStore/index.ts]: @OR13 why are we commending out the cool command line interface

File: src/CLI/EventStore/spec.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
+
+// const {
```
Comment (eolszewski) [inline src/CLI/EventStore/spec.ts]: @OR13 why are we commenting this out

File: src/CLI/Patch/commands/moveDirectory.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (eolszewski) [inline src/CLI/Patch/commands/moveDirectory.ts]: This is much better than having both moveTests and moveContracts. 👍

File: src/CLI/TransmuteIpfs/spec.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
+
+
+// const {
```
Comment (eolszewski) [inline src/CLI/TransmuteIpfs/spec.ts]: @OR13 why are we commenting this out?

**Pull request merged**